src/Data/ProgrammesDb/Entity/Image.php

```diff
@@ -107,4 +121,24 @@ public function setExtension(string $extension): void
     {
         $this->extension = $extension;
     }
+
+    public function getHeight(): ?int
+    {
+        return $this->height;
+    }
+
+    public function setHeight(int $height): void
```

Comment:
For setting height and width, should the params be nullable integers? I'm not sure if PIPs always includes the dimensions

Reply:
Ahh yes they should! Good catch! Updated now 😄